service/src/test/java/uk/gov/hmcts/reform/fpl/service/MigrateCFVServiceTest.java

```diff
@@ -1500,6 +1528,325 @@ void courtBundlesShouldBeInSameListAfterRollback() {
+            assertThat(updatedFields).doesNotContainKey("correspondenceDocListCTSC");
+
+            assertThat(updatedFields).extracting("archivedDocumentsListCTSC").asList()
+                .containsExactlyInAnyOrder(
+                    element(docOneId, ManagedDocument.builder().document(document1).build()),
+                    element(docTwoId, ManagedDocument.builder().document(document2).build()),
+                    element(docThreeId, ManagedDocument.builder().document(document3).build()),
+                    element(docFourId, ManagedDocument.builder().document(document4).build()));
+        }
+
+        @Test
+        void shouldMigrateApplicationDocumentsToArchivedDocuments() {
+            UUID docOneId = randomUUID();
+            UUID docTwoId = randomUUID();
+            UUID docThreeId = randomUUID();
+            UUID docFourId = randomUUID();
+            DocumentReference document1 = DocumentReference.builder().build();
+            DocumentReference document2 = DocumentReference.builder().build();
+            DocumentReference document3 = DocumentReference.builder().build();
+            DocumentReference document4 = DocumentReference.builder().build();
+
+            Element<ApplicationDocument> docOne = element(docOneId,
+                ApplicationDocument.builder().document(document1).build());
+            Element<ApplicationDocument> docTwo = element(docTwoId,
+                ApplicationDocument.builder().document(document1).build());
+            Element<ApplicationDocument> docThree = element(docThreeId,
+                ApplicationDocument.builder().document(document1).build());
+            Element<ApplicationDocument> docFour = element(docFourId,
+                ApplicationDocument.builder().document(document1).build());
+
+            CaseData caseData = CaseData.builder()
+                .id(1L)
+                .applicationDocuments(List.of(docOne, docTwo, docThree, docFour))
+                .build();
+
+            Map<String, Object> updatedFields = underTest.migrateApplicationDocumentsToArchivedDocuments(caseData);
+            List.of("documentsFiledOnIssueList", "documentsFiledOnIssueListLA",
+                "documentsFiledOnIssueListCTSC",
+                "carePlanList", "carePlanListLA", "carePlanListCTSC",
+                "thresholdList", "thresholdListLA", "thresholdListCTSC").stream().forEach(f -> assertThat(updatedFields)
+                    .doesNotContainKey(f));
+
+            assertThat(updatedFields).extracting("archivedDocumentsListCTSC").asList()
+                .containsExactlyInAnyOrder(
+                    element(docOneId, ManagedDocument.builder().document(document1).build()),
+                    element(docTwoId, ManagedDocument.builder().document(document2).build()),
+                    element(docThreeId, ManagedDocument.builder().document(document3).build()),
+                    element(docFourId, ManagedDocument.builder().document(document4).build()));
+        }
```

Comment:
In the above tests, docOne,two,three,four are all built using document 1.  Is this correct?

Reply:
Good spot. I am fixing them.